<!--Coverage Node launch file-->
<launch>
	<node name="coverage_map_node" type="coverage_map_publisher.py" pkg="my_2d_nav" output="screen"/>
	<rosparam file="$(find my_2d_nav)/move_base_config/params.yaml" command="load"/>
</launch>
